dina-base-api/src/test/java/ca/gc/aafc/dina/security/DinaRepoPermissionMetaTest.java

```diff
@@ -75,18 +77,18 @@ void setUp() {
   }
```

Comment:
Changes made to this file:

* The "read" permission should now be returned when checking the permissions as an admin.
* Removed { } in @WithMockKeycloakUser annotation which was causing a checkstyle violation.
* TYPE_NAME is now public like the rest of the TYPE_NAME variables.
* Added the new authorizeRead method.